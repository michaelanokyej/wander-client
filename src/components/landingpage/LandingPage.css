.landing-page {
  background: url("./landingPageBG.jpg");
  background-size: cover;
}
