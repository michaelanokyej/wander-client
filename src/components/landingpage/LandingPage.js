import React from "react";
import About from "./about/About";
import HomeTourList from "../home-tour-list/HomeTourList";
import "./LandingPage.css";

class Homepage extends React.Component {
  render() {
    return (
      <main className="landing-page">
        <div>
          <h1>Wander</h1>
          <h4>Tour like a local</h4>
        </div>
        <About />
        <HomeTourList />
      </main>
    );
  }
}

export default Homepage;
