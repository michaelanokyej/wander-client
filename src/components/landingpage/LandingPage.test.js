import React from "react";
import LandingPage from "./LandingPage"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

describe(`LandingPage component`, () => {
    it(`renders the LandingPage component`, () => {
        const wrapper = shallow(<LandingPage />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});