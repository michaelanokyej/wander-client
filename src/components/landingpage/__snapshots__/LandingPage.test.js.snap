// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage component renders the LandingPage component 1`] = `
<main
  className="landing-page"
>
  <div>
    <h1>
      Wander
    </h1>
    <h4>
      Tour like a local
    </h4>
  </div>
  <About />
  <HomeTourList />
</main>
`;
